Resources/translations/SonataAdminBundle.en.xliff

```diff
@@ -162,6 +162,10 @@
               <source>flash_edit_error</source>
               <target>An error has occurred during update of item "%name%".</target>
             </trans-unit>
+            <trans-unit id='flash_lock_error'>
+                <source>flash_lock_error</source>
+                <target>Another user has modified item "%name%". Please reload the page and apply the changes again.</target>
```

Comment:
I think the message should be: Another user has modified item "%name%". Please <a href="...">click here</a> to reload the page and apply the changes again. Maybe we can use some font awesome icon too.